ChemistryLib/PhreeqcIOData/EquilibriumPhase.cpp

```diff
@@ -13,15 +13,10 @@
 
 namespace ChemistryLib
 {
-std::ostream& operator<<(std::ostream& os,
-                         EquilibriumPhase const& equilibrium_phase)
+void EquilibriumPhase::print(std::ostream& os,
+                             std::size_t const& chemical_system_id) const
```

Comment:
pass small things by value.

Reply:
Not fixed yet, as we may have better idea to retain the operator overloading function.

Reply:
Fixed.